features/step_definitions/helper_services.rb

```diff
@@ -379,8 +379,8 @@
     raise "project #{project_name} does not exist"
   end
 
-  @result = user.cli_exec(:create, f: "#{BushSlicer::HOME}/testdata/routing/header-test/dc.json")
-  raise "could not create header test dc" unless @result[:success]
+  @result = user.cli_exec(:create, f: "#{BushSlicer::HOME}/testdata/routing/header-test/deployment.yaml")
+  raise "could not create header test deployment" unless @result[:success]
   cb.header_test_dc = dc("header-test")
```

Comment:
Should we update this line `dc("header-test")` as well ?

Reply:
yes, updated to deployment